### DISCUSSION TITLE
Why does Symfony recipe support old directory structure by default?

### DISCUSSION THREAD

**Discussion by brooksvb** [deployphp/deployer#3000] [General]
https://github.com/deployphp/deployer/blob/28534235ede6407ad2d59067a1f326f3d430d936/recipe/symfony.php#L45-L47

Using the Symfony recipe, I've had to override a bunch of options for a Symfony 4/5 installation. It doesn't make sense that the recipe has defaults that are for an old version; like the console path should be `bin/console` by default, and the var directory should be `var` by default, with `cache` and `log` inside.

Am I under some wrong impressions here?

**Comment by antonmedv** [comment]
https://github.com/deployphp/deployer/blob/master/recipe/symfony.php
